package.json

```diff
@@ -95,62 +95,62 @@
   "entry": "lib/index.js",
   "types": "lib/index.d.ts",
   "devDependencies": {
-    "@chanzuckerberg/axe-storybook-testing": "^6.1.0",
-    "@chanzuckerberg/eslint-config-edu-js": "^1.0.1",
-    "@chanzuckerberg/eslint-config-edu-ts": "^1.0.0",
-    "@chanzuckerberg/eslint-plugin-edu-react": "^1.1.0",
+    "@chanzuckerberg/axe-storybook-testing": "^6.2.0",
+    "@chanzuckerberg/eslint-config-edu-js": "^1.0.2",
+    "@chanzuckerberg/eslint-config-edu-ts": "^1.0.4",
+    "@chanzuckerberg/eslint-plugin-edu-react": "^1.1.7",
     "@chanzuckerberg/eslint-plugin-stories": "^3.2.2",
```

Comment:
Latest is 3.2.7 but the `no-components-without-story` rule flags components with a story file extension of `TS`, will wait until https://github.com/chanzuckerberg/frontend-libs/pull/41 and new publish